src/client/components/equalizer/equalizer.js

```diff
@@ -76,140 +55,138 @@ class Equalizer extends Component {
       navigator.mediaDevices.getUserMedia({
         audio: true,
       })
-        .then(stream => this.createSoundStream(stream))
-        .catch((e) => {
-          throw new Error(e);
+        .then(
+          this.createSoundStream(),
+        )
+        .catch((errorStream) => {
+          throw new Error(errorStream);
         });
     } else {
       throw new Error('browser doesnt support audio API');
     }
   }
 
-  widthMerge = (eventFromInputTypeRange) => {
-    const { mergeCanvasWidth: mergeCanvasWidthAsProp } = this.props;
-    mergeCanvasWidthAsProp(eventFromInputTypeRange);
-  }
-
-  playSoundFromFile = () => {
+  uploadSoundInfoFromFile = (e) => {
+    const [file] = e.target.files;
+    const audioFile = new Audio(URL.createObjectURL(file));
+    const sound = new Pizzicato.Sound({
+      source: 'file',
+      options: {
+        path: audioFile.src,
+        loop: true,
+      },
+    }, () => this.createSoundInfoInState(sound, file));
+  };
+
+  createSoundInfoInState = (sound, file) => {
     const { audioData } = this.props;
-    const { audioFromFile: soundFromFile, playPauseState } = audioData;
-    if (!playPauseState) {
-      soundFromFile.play();
-      this.equaliserRun();
-    } else {
-      soundFromFile.pause();
-    }
-    const { playPauseSoundFromFile: playPauseSoundFromFileAsProp } = this.props;
-    playPauseSoundFromFileAsProp();
+    const { analyser } = audioData;
+    sound.connect(analyser);
+    const { createAudioData: createAudioDataAsProp } = this.props;
```

Comment:
Are you actually need to rename this prop?

Reply:
I forgot that I get this from lint: "error  'createStreamData' is already declared in the upper scope" - must renamed all back(

Reply:
So you need to use only one name for one function, not one name for lots of functions

Reply:
I imported 4 functions (createAudioData,  playPauseSoundFromFile,  createStreamData,  startMuteStreamAudio) and use them as mapDispatchToProps (actions) in method connect. And linter does not allow me to use the same name.